docs/proposals/web_uploads.md

```diff
@@ -0,0 +1,34 @@
+# Add upload scheduler to webservice
+
+This is a proposal to facet `bundle upload` via the Datamon webservice (i.e. the `web` command).
+
+Additionally, so that the web service -- currently a frontend for `repo list`, `bundle list`, and `bundle list files`, all operations that typically complete in per-HTTP-request goroutines in about the expected timeframe for page loads in a 1.0-ish webservice -- can continue to be useful for a possible deploy to Kubernetes, faceting `bundle upload` necessarily includes implementing some at least rudimentary scheduling functionality:  The webservice will need to keep track of several ongoing uploads, each triggered by an HTTP-request, but with fire-and-forget (i.e. event) semantics rather than those of request-response (i.e. RPC).
```

Comment:
An alternative could be python-based SDK "async bundle upload"

Reply:
Another way, that I am developing a bit below, could be an enhanced mutable mount feature.
At the moment, the mutable mount is only experimental, still has some bugs, and overall probably not used at all.

This could change by building on top of it some more user-friendly features like the one described in this paper.

Reply:
> async bundle upload

whether we implement in Python or not, i am indeed viewing this more in the async/coroutine style: heartbeats are conceptually adjacent to the event loop that coroutines usually are based on.

> could be an enhanced mutable mount feature

yerp, there are lots of ways to translate the motivation in the description of this merge request to functionality to functionality that might or might not eventually get used to assuage the essential pain point (real or perceived).